components/dropdown/stories/dropdown.stories.ts

```diff
@@ -0,0 +1,652 @@
+<auro-dropdown aria-label="custom label">
+  Lorem ipsum solar
+  <div slot="trigger">
+    Trigger
+  </div>
+</auro-dropdown>
+  `
+};
+
+export const BasicIcon: Story = {
+  render: () => html`
+<auro-dropdown aria-label="custom label">
+  Lorem ipsum solar
+  <div slot="trigger">
+    <auro-icon
+      category="interface"
+      name="arrow-down"></auro-icon>
+  </div>
+</auro-dropdown>
+  `
+};
+
+export const BasicButton: Story = {
+  render: () => html`
+<auro-dropdown aria-label="custom label">
+  Lorem ipsum solar
+  <div slot="trigger">
+    <auro-button
+      slot="trigger"
+      fluid>
+      Dropdown
+    </auro-button>
+  </div>
+</auro-dropdown>
+  `
+};
+
+export const Bordered: Story = {
+  render: () => html`
+<auro-dropdown aria-label="custom label" bordered>
+  Lorem ipsum solar
+  <div slot="trigger">
+    Trigger
+  </div>
+</auro-dropdown>
+  `
+};
+
+export const Chevron: Story = {
+  render: () => html`
+<auro-dropdown aria-label="custom label" chevron>
+  Lorem ipsum solar
+  <div slot="trigger">
+    Trigger
+  </div>
+</auro-dropdown>
+  `
+};
+
+export const ChevronIcon: Story = {
+  render: () => html`
+<auro-dropdown aria-label="custom label" chevron>
+  Lorem ipsum solar
+  <div slot="trigger">
+    <auro-icon
+      category="interface"
+      name="arrow-down"></auro-icon>
+  </div>
+</auro-dropdown>
+  `
+};
+
+export const ChevronButton: Story = {
+  render: () => html`
+<auro-dropdown aria-label="custom label" chevron>
+  Lorem ipsum solar
+  <div slot="trigger">
+    <auro-button
+      slot="trigger"
+      fluid>
+      Dropdown
+    </auro-button>
+  </div>
+</auro-dropdown>
+  `
+};
+
+export const ChevronInput: Story = {
+  render: () => html`
+<auro-dropdown aria-label="custom label" chevron>
+  Lorem ipsum solar
+  <div slot="trigger">
+    <auro-input
+      slot="trigger"
+      id="inputExampleTrigger">
+    </auro-input>
+  </div>
+</auro-dropdown>
+  `
+};
+
+export const Disabled: Story = {
+  render: () => html`
+<auro-dropdown aria-label="custom label" disabled>
+  Lorem ipsum solar
+  <div slot="trigger">
+    Trigger
+  </div>
+</auro-dropdown>
+  `
+};
+
+export const DisabledAll: Story = {
+  render: () => html`
+<auro-dropdown
+  aria-label="custom label"
+  disabled
+  chevron
+  rounded
+  inset
+  bordered>
+  Lorem ipsum solar
+  <div slot="trigger">
+    Trigger
+  </div>
+  <span slot="helpText">
+    Helper text
+  </span>
+  <span slot="label">
+    Name
+  </span>
+</auro-dropdown>
+  `
+};
+
+export const Error: Story = {
+  render: () => html`
+<auro-dropdown aria-label="custom label" error>
+  Lorem ipsum solar
+  <div slot="trigger">
+    Trigger
+  </div>
+</auro-dropdown>
+  `
+};
+
+export const ErrorBordered: Story = {
+  render: () => html`
+<auro-dropdown
+  aria-label="custom label"
+  inset
+  error
+  rounded
+  bordered>
+  Lorem ipsum solar
+  <div slot="trigger">
+    Trigger
+  </div>
+</auro-dropdown>
+  `
+};
+
+// TODO: Confirm functionality
```

Comment:
Sometimes the purpose of an example wasn't clear to me, so I couldn't confirm whether it was working or not.